CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.0 (2018-8-28)
+
+- Added icon styles for sky-switch-control form elements
```

Comment:
@blackbaud-conorwright Mind putting a link to the pull request in there? Also, needs a period at the end of the sentence.

Reply:
Should `sky-switch-control` also be formatted as a code snippet?

Reply:
done